terraform-module/modules/frontend-spa-dns/main.tf

```diff
@@ -38,3 +38,15 @@ resource "aws_route53_record" "dns_record" {
     evaluate_target_health = false
   }
 }
+
+# Based on request SRE-7285 to provide better demo experience
+# Creates ${app_name}.demo.pleo.io record that redirects to the staging env
+# Records are created on product-production env, because we need to evaluate at pleo.io level (not staging.pleo.io).
+resource "aws_route53_record" "demo_dns_record" {
+  for_each = lower(var.env) == "production" ? local.naked : {}
+
+  name    = "${var.app_name}.demo.${var.zone_domain}"
```

Comment:
just an idea: wouldn't it be nicer to have `demo.pleo.io` redirecting to `app.staging.pleo.io`, dropping the `app` subdomain?

Reply:
I meant exactly what you describe as first option actually.
story.staging.pleo.io is just a playground for frontend components, this is super useful for internal demos but nothing we would share internally. telescope.staging.pleo.io is just the docs for the design system, also nothing that needs an external demo environment.